docs/source/contributing/developing.adoc

```diff
@@ -315,6 +315,20 @@ You can correct the formatting errors yourself or instruct the violations to be
 $ make fmt
 ----
 
+[[commit-messages]]
+=== Commit messages
+
+Each commit message messages needs meet two requirements:
```

Comment:
Repeated "message messages", should be "message".

Also, "needs meet" is grammatically incorrect (should be "needs to meet") -- I also think that the wording is more concise and clear if we replace "needs to" with "must":

"Each commit message must meet two requirements:"

Reply:
Thanks, good catch.